com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -43,16 +44,7 @@ The version number for this package has increased due to a version update of a r
 - Fixed MSAA on Metal MacOS and Editor.
 - Fixed an issue causing passthrough camera to not render. [case 1283894](https://issuetracker.unity3d.com/product/unity/issues/guid/1283894/) 
 - Fixed MSAA override on camera does not work in non-XR project if target eye is selected to both eye.
-- Fixed a compiler error in BakedLit shader when using Hybrid Renderer.
-- Fixed XR camera fov can be changed through camera inspector.
-- Fixed LightProbes to have gamma correct when using gamma color space. [case 1268911](https://issuetracker.unity3d.com/issues/urp-has-no-gamma-correction-for-lightprobes)
-- Fixed an issue causing materials to be upgraded multiple times.
-- Fixed an issue where the Camera inspector was grabbing the URP asset in Graphics Settings rather than the currently active.
-- Fixed an issue where the Light Explorer was grabbing the URP asset in Graphics Settings rather than the currently active.
-- Fixed an issue where the scene view camera ignored the pipeline assets HDR setting. [case 1284369](https://issuetracker.unity3d.com/issues/urp-scene-view-camera-ignores-pipeline-assets-hdr-settings-when-main-camera-uses-pipeline-settings)
-- Fixed bloom inconsistencies between Gamma and Linear color-spaces.
-- Fixed an issue with upgrading material set to cutout didn't properly set alpha clipping. [case 1235516](https://issuetracker.unity3d.com/issues/urp-upgrade-material-utility-does-not-set-the-alpha-clipping-when-material-was-using-a-shader-with-rendering-mode-set-to-cutout)
-- Fixed issue where selecting and deselecting Forward Renderer asset would leak memory [case 1290628](https://issuetracker.unity3d.com/issues/urp-scriptablerendererfeatureeditor-memory-leak-while-interacting-with-forward-renderer-in-the-project-window)
+- Fixed an issue where Universal Render Pipeline with disabled antiAliasing was overwriting QualitySettings.asset on frequent cases. [case 1219159](https://issuetracker.unity3d.com/issues/urp-qualitysettings-dot-asset-file-gets-overwritten-with-the-same-content-when-the-editor-is-closed)
```

Comment:
Merge error? This seems duplicate from above and why removing the other entries?